Wikipedia/Code/WMFArticleContainerViewController.m

```diff
@@ -556,13 +556,17 @@ - (void)viewWillAppear:(BOOL)animated {
     }
 }
 
+- (void)viewDidAppear:(BOOL)animated {
+    [super viewDidAppear:animated];
+    [[NSUserDefaults standardUserDefaults] wmf_setOpenArticleTitle:self.articleTitle];
```

Comment:
did you check if this cause problems w/ peeking "erasing" the previous open article?